src/DataTrail.tsx

```diff
@@ -302,7 +304,8 @@ export class DataTrail extends SceneObjectBase<DataTrailState> implements SceneO
     const { controls, topScene, settings, pluginInfo, embedded } = model.useState();
 
     const chromeHeaderHeight = useChromeHeaderHeight();
-    const styles = useStyles2(getStyles, embedded ? 0 : chromeHeaderHeight ?? 0);
+    const headerHeight = embedded ? 0 : chromeHeaderHeight ?? 0;
```

Comment:
```suggestion
    const chromeHeaderHeight = embedded ? 0 : useChromeHeaderHeight();
```

Reply:
first time using the suggestion feature but thats the idea (in case it keeps both lines which is not the suggestion)

Reply:
+1, I think we need to call `useChromeHeaderHeight()` non-conditionally to prevent React errors

To avoid the nested ternary, we could do this:
```tsx
const chromeHeaderHeight = useChromeHeaderHeight() ?? 0;
const headerHeight = embedded ? 0 : chromeHeaderHeight;
```

Reply:
done in https://github.com/grafana/metrics-drilldown/pull/558/commits/955bf3cb14bc68a02ca2237b883f5e5c2e39fbbc